packages/js/onboarding/changelog/9063-add-shared-logos-component

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Added a new shared component for WooPayments’ payment method logos.
```

Comment:
I noticed in the [WooCommerce changelog](https://github.com/woocommerce/woocommerce/blob/trunk/changelog.txt) that we like to keep sentences in present tense, like git commit messages.

```suggestion
Add a new shared component to display logos of payment methods supported by WooPayments.
```

Reply:
Updated!